File: mobile_apps/IOS/deliveryman/Edelivery Provider/Models/UserDataResponse/OrderHistoryResponse.swift


import Foundation
 
public class OrderHistoryResponse {
	public var success : String?
	public var message : Int?
	public var order_list : Array<Order_list>?


    public class func modelsFromDictionaryArray(array:NSArray) -> [OrderHistoryResponse] {
        var models:[OrderHistoryResponse] = []
        for item in array {
            models.append(OrderHistoryResponse(dictionary: item as! [String:Any])!)
        }
        return models
    }


	required public init?(dictionary: [String:Any]) {

		success = dictionary["success"] as? String
		message = dictionary["message"] as? Int
		if (dictionary["request_list"] != nil) {
            order_list = Order_list.modelsFromDictionaryArray(array: dictionary["request_list"] as! NSArray) }
	}


	public func dictionaryRepresentation() -> [String:Any] {

		 var dictionary:[String:Any] = [:]
        dictionary["success"] = self.success
        dictionary["message"] = self.message
        return dictionary
	}

}

File: mobile_apps/Android/deliveryman/dropoprovider/src/main/java/com/dropo/provider/models/responsemodels/OrderHistoryResponse.java
package com.dropo.provider.models.responsemodels;

import com.dropo.provider.models.datamodels.OrderHistory;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class OrderHistoryResponse {

    @SerializedName("success")
    private boolean success;

    @SerializedName("message")
    private int message;
    @SerializedName("status_phrase")
    private String statusPhrase;
    @SerializedName("request_list")
    private List<OrderHistory> orderList;
    @SerializedName("error_code")
    @Expose
    private int errorCode;

    public boolean isSuccess() {
        return success;
    }

    public void setSuccess(boolean success) {
        this.success = success;
    }

    public int getMessage() {
        return message;
    }

    public void setMessage(int message) {
        this.message = message;
    }

    public String getStatusPhrase() {
        return statusPhrase;
    }

    public void setStatusPhrase(String statusPhrase) {
        this.statusPhrase = statusPhrase;
    }

    public List<OrderHistory> getOrderList() {
        return orderList;
    }

    public void setOrderList(List<OrderHistory> orderList) {
        this.orderList = orderList;
    }

    public int getErrorCode() {
        return errorCode;
    }

    public void setErrorCode(int errorCode) {
        this.errorCode = errorCode;
    }
}